### PR TITLE
Longest Increasing Subsequence (c#)

### PR DESCRIPTION
**Pull request by biekymahato**
Fixes #2253 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Languages Used:
- c#

#### Files Added:
- Longest Increasing Subsequence (c#)

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by biekymahato** [comment]
@asha15  Have a look.

**Comment by biekymahato** [comment]
please do review @jainaman224 @asha15

File: Longest Increasing Subsequence (c#)/Longest_Increasing_Subsequence.cs [hunk 1]
```diff
@@ -0,0 +1,59 @@
+/* Dynamic Programming C# implementation of LIS problem */
+using System ;
+class LIS
```
Comment (singh-shreya6) [inline Longest Increasing Subsequence (c#)/Longest_Increasing_Subsequence.cs]: Please add a blank line before this.

File: Longest Increasing Subsequence (c#)/Longest_Increasing_Subsequence.cs [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        for(int i = 0; i < n; i++)
+            arr[i] = Convert.ToInt32(Console.ReadLine());
+
+        Console.WriteLine("Length of lis is " + lis( arr, n ) + "\n" );
+    }
+}
+
+/*
+Input :
+8    //n = 8
```
Comment (singh-shreya6) [inline Longest Increasing Subsequence (c#)/Longest_Increasing_Subsequence.cs]: why do you have // inside multiline comment.

File: Longest Increasing Subsequence (c#)/Longest_Increasing_Subsequence.cs [hunk 3]
```diff
@@ -0,0 +1,59 @@
+/* Dynamic Programming C# implementation of LIS problem */
```
Comment (singh-shreya6) [inline Longest Increasing Subsequence (c#)/Longest_Increasing_Subsequence.cs]: Please add a brief description about what is LIS.

**Pull request closed without merging**